src/ServicePulse.Host/app/lib/bootstrap-3.3.5/css/bootstrap.css

```diff
@@ -1583,8 +1583,8 @@ pre code {
   overflow-y: scroll;
 }
 .container {
-  padding-right: 15px;
-  padding-left: 15px;
+  padding-right: 0;
+  padding-left: 0;
```

Comment:
@sergioc is there any way for us to not change bootstrap.css? For example to override this value in our particular.css file? If we upgrade bootstrap those changes will be gone.